R/mht01.R

```diff
@@ -105,14 +105,16 @@ mht01_pre <- function(adam_db, ...) {
 #' @returns the postprocessing function returns an `rtables` object or an `ElementaryTable` (null report).
 #' @export
 #'
-mht01_post <- function(tlg, prune_0 = TRUE, ...) {
+mht01_post <- function(tlg, row_split_var = "MHBODSYS", prune_0 = TRUE, ...) {
   if (prune_0) {
     tlg <- smart_prune(tlg)
   }
 
+  row_split_var <- c(rbind(row_split_var, "*"))
+
   tbl_sorted <- tlg %>%
     sort_at_path(
-      path = c("MHBODSYS", "*", "MHDECOD"),
+      path = c(row_split_var, "MHDECOD"),
```

Comment:
should be split1, *, split 2, *, split 3, * mhdecod?

Reply:
I added the `row_split_var <- c(rbind(row_split_var, "*"))` just above to do that. It is a bit funny but apparently the shortest way to alternate elements from two different vectors.